src/encoder/encoderfdkaac.cpp

```diff
@@ -261,10 +261,10 @@ int EncoderFdkAac::initEncoder(int samplerate, QString* pUserErrorMessage) {
             // TODO(Palakis): write installation guide on Mixxx's wiki
             // and include link in message below
             *pUserErrorMessage = QObject::tr(
-                    "<html>Mixxx cannot record or stream in AAC "
+                    "Mixxx cannot record or stream in AAC "
```

Comment:
Are you sure this isn't to force rich formatting?

Reply:
This also applies to 2.3. If we fix it we should fix it there first.